File: Secao07 - Comandos de repetição/Exercicios em Python/Exercicio05.py
# -*- coding: utf-8 -*-
nome = input("Informe seu usuario: ")
senha = input("Informe sua senha: ")

if senha == nome:
    print("Erro! Senha não pode ser igual ao nome.\n")
    nome = input("Informe seu usuario: ")
    senha = input("Informe sua senha: ")
else:
    print("Parabens! Usuario Logado.")

File: Secao07 - Comandos de repetição/Exercicios em Java/Exercicio05.java
package br.com.udemy.secao07;

import java.util.Scanner;

public class Exercicio05 {

	public static void main(String[] args) {
		Scanner teclado = new Scanner(System.in);
		String nome, senha;
		
		System.out.print("Informe seu usuario: ");
		nome = teclado.next();
		System.out.print("Informe sua senha: ");
		senha = teclado.next();
		
		
		while(nome.equals(senha)) {
			System.out.println("Erro! Nome não pode ser igual a senha.");
			System.out.println(" ");
			
			System.out.print("Informe seu usuario: ");
			nome = teclado.next();
			System.out.print("Informe sua senha: ");
			senha = teclado.next();
		}
		teclado.close();
	}

}
